### PR TITLE
Create schema, 4queries, add seed data file, and doctype into pug files

### PR DESCRIPTION
**Pull request by sashajustice**
Here is what I have so far

### CODE REVIEW

File: src/database/db.js [hunk 1]
```diff
@@ -11,4 +11,43 @@ if(process.env.NODE_ENV === 'development'){
...
+
+
+
+
+
+
+
+
+
+module.exports = db, getAllTodos, getCompletedTodos,completeOneTodo,addOneTodo   ;
```
Comment (DianaVashti) [inline src/database/db.js]: don't need to export `db` here, only the functions, and wrap them up in some curly boys...
`{getAllTodos, getCompletedTodos, completeOneTodo, addOneTodo}`

Comment (DianaVashti) [reply src/database/db.js]: also the specs said to add these functions to the actions folder, either as one actions.js file or as separate files for each function

File: src/database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+DROP DATABASE IF EXISTS todos;
+CREATE DATABASE todo;
```
Comment (DianaVashti) [inline src/database/schema.sql]: the scripts this project came with do the dropping and creating of the data base so remove these two lines and only create tables in this file.

File: src/database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+\c todos
+
+CREATE TABLE todos (
+id SERIAL PRIMARY KEY,
+description TEXT NOT NULL,
+is_completed BOOLEAN
+);
```
Comment (DianaVashti) [inline src/database/schema.sql:5]: looks good!

File: src/database/seed.sql [hunk 4]
```diff
@@ -0,0 +1,7 @@
+\c todos
+
+INSERT INTO todos (name, description) VALUES
+('Groceries', '10 items')
+('Clean Room', 'Storage')
+('Meditate', '20Min')
+('Jobs', 'Apply to 20')
```
Comment (DianaVashti) [inline src/database/seed.sql]: The table above only has a description field not a name, so either change the schema or remove one of the fields from the seed data

**Review by DianaVashti — COMMENTED** [review]
Good start, looking good. Review the comments and shoot me questions if you need some clarification

File: src/database/db.js (hunk 1 above)
Comment (sashajustice) [reply src/database/db.js]: All checks have been resolved. As far as actions folder there isn't one but i've created on in the src folder.

**Pull request closed without merging**